tests/page-objects/wizard/FilterEditor.ts

```diff
@@ -23,29 +23,30 @@ export default class FilterEditor {
     }
 
     async selectDate(dateValue: string) {
-        await this.setInputValue(dateValue);
-        await this.closeDatepickerPopup();
+        await fillDatePicker({
+            page: this.page,
+            selector: '.dl-dialog-filter__body .g-text-input__control',
```

Comment:
Maybe it will be possible to redo it on `getByRole`?
So as not to get tied up in internal classes